DependencyInjection/EzWorkflowEngineExtension.php

```diff
@@ -15,14 +17,27 @@
 class EzWorkflowEngineExtension extends Extension
 {
     /**
-     * {@inheritDoc}
+     * {@inheritdoc}
      */
+    public function getAlias()
+    {
+        return "ez_workflow_engine";
+    }
+
+
     public function load(array $configs, ContainerBuilder $container)
     {
         $configuration = new Configuration();
-        $config = $this->processConfiguration($configuration, $configs);
+        $config        = $this->processConfiguration( $configuration, $configs );
+
+        $loader = new Loader\YamlFileLoader( $container, new FileLocator( __DIR__ . '/../Resources/config' ) );
+        $loader->load( 'services.yml' );
+        $loader->load( 'default_settings.yml' );
 
-        $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
-        $loader->load('services.yml');
+        $processor = new ConfigurationProcessor( $container, 'ez_workflowengine_bundle' );
```

Comment:
I am not sure that I fully understand this.
I my mind, either we have the users put parameters in config files (1), or we have the users put semantic-config in config files, and then transform that config into services or parameters (2).

It seems here that you went for approach 1, which should work well enough for the specified needs (having the config change per-siteaccess), as long as users use the correct format for the parameter name.

The usage of the ConfigurationProcessor instead seems to point to option 2, where we want the users to define the setting using semantic-configuration format, and have this class create the value of the settings.
But in this case:
- why not document the available semantic config and how to set it up ?
- why create the default_settings.yml file ?